debian/rules

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/make -f
 # -*- makefile -*-
-VERSION := $(shell mvn org.apache.maven.plugins:maven-help-plugin:2.1.1:evaluate -Dexpression=project.version | grep -v "\[")
+VERSION := $(shell awk 'f{print;f=0} /<artifactId>cloudstack/{f=1}' pom.xml| sed -e 's/<[^>]*>//g'  -e 's/^[ \t]*//')
```

Comment:
@rsafonseca I appreciate the optimisation, but given that we're not building deb packages (at least  locally often, we've jenkins for that) and with respect to readability of the code I would prefer to keep the mvn based version fetching logic that you had done earlier -- to me that makes more sense and is readable.

I think you may close this one, though I'm willing to merge it if you strongly need this, for example building deb packages locally often.